taskfiles/Taskfile_systemtest.yml

```diff
@@ -0,0 +1,34 @@
+version: '3'
+
+tasks:
+  preflight:
```

Comment:
Can we use the standard "task test:setup" nomenclature for this?

https://jitsuin.atlassian.net/wiki/spaces/ENG/pages/3140157463/Common+Tasks+Across+All+Repos

Reply:
done